bin/main.rb

```diff
@@ -0,0 +1,137 @@
+require '../lib/library.rb'
+class Game
```

Comment:
Please try and have this Game class in on file and move it to lib, may have to reconsider some methods and how you implement them. If you need tips, please message me ASAP! I am glad to help!